relay-dynamic-config/src/metrics.rs

```diff
@@ -114,7 +114,8 @@ pub struct CustomMeasurementConfig {
 ///      - Emit a `usage` metric and use it for rate limiting.
 ///      - Delay metrics extraction for indexed transactions.
 ///  - 4: Adds support for `RuleConfigs` with string comparisons.
-const TRANSACTION_EXTRACT_VERSION: u16 = 4;
+///  - 5: No change, bumped together with [`MetricExtractionConfig::MAX_SUPPORTED_VERSION`].
+const TRANSACTION_EXTRACT_VERSION: u16 = 5;
```

Comment:
If we do not bump this version as well, external Relays will still perform metrics extraction from transactions, but skip the span metrics. See https://github.com/getsentry/relay/blob/89de2d1ee6819f26fee76df204b638159e5b42a3/relay-server/src/services/processor.rs#L1159-L1166